database/migrations/2020_09_02_132413_update_users_table.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class UpdateUsersTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('users', function (Blueprint $table) {
+            $table->string('profile_img')->nullable()->change();
+            $table->boolean('is_active')->default(true)->change();
+        });
+    }
+
+    /**
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down()
+    {
+        Schema::table('users', function (Blueprint $table) {
+            //
```

Comment:
I know it's annoying but try to add a rollback command. 
in the first column you can us this command: 
$table->string('profile_img')->nullable(false)->change();
in the second column use this command: 
$table->boolean('is_active')->default(false)
Rolling back and forth is very important to give us control on the whole application.

Reply:
resolved [here](https://github.com/BuildForSDGCohort2/Team-248-Backend/pull/29/commits/00875fe6132217e2cc913f10f38da16976ad6748)

btw I don't see the importance of this change